src/lib/models/event-history/index.ts

```diff
@@ -12,7 +17,11 @@ export async function getEventAttributes(
   historyEvent: HistoryEvent,
 ): Promise<EventAttributesWithType> {
   const { key, attributes } = findAttributesAndKey(historyEvent);
-  const decodedAttributes = await convertPayloadToJson(attributes);
+  const port = get(dataConverterPort);
+
+  const decodedAttributes = port
+    ? await convertPayloadToJsonWithWebsocket(attributes)
+    : await convertPayloadToJsonWithCodec(attributes);
```

Comment:
What if the remote codec and the websocket are configured? Should we show an error? Or provide some ux to set it to the other option?

Reply:
Good question. I'll ask Candace.